index.bs

```diff
@@ -801,27 +809,32 @@ authorizing an authenticator with which to complete the operation.
 
-    - The <dfn>timeout</dfn> parameter specifies a time, in milliseconds, that the caller is willing to wait for the call to
-        complete. This is treated as a hint, and may be overridden by the platform.
+    - The optional <dfn>timeout</dfn> parameter specifies a time, in milliseconds, that the caller is willing to wait for the call
+        to complete. This is treated as a hint, and may be overridden by the platform.
 
-    - The <dfn>rpId</dfn> parameter explicitly specifies the RP ID that the credential should be associated with. If it is
-        omitted, the RP ID will be set to the [=origin=] specified by the {{WebAuthentication}} object's [=relevant settings
+    - The optional <dfn>rpId</dfn> parameter explicitly specifies the RP ID that the credential should be associated with. If it 
+        is omitted, the RP ID will be set to the [=origin=] specified by the {{WebAuthentication}} object's [=relevant settings
         object=].
 
-    - The <dfn>excludeList</dfn> parameter is intended for use by [=[RPS]=] that wish to limit the creation of multiple
+    - The optional <dfn>excludeList</dfn> parameter is intended for use by [=[RPS]=] that wish to limit the creation of multiple
         credentials for the same account on a single authenticator. The platform is requested to return an error if the new
         credential would be created on an authenticator that also contains one of the credentials enumerated in this parameter.
 
-    - The <dfn>extensions</dfn> parameter contains additional parameters requesting additional processing by the client and
-        authenticator. For example, the caller may request that only authenticators with certain capabilities be used to create
-        the credential, or that particular information be returned in the [=attestation object=]. The caller may also specify
-        an additional message that they would like the authenticator to display to the user. Extensions are defined in
-        [[#extensions]].
-
-    - The <dfn>attachment</dfn> parameter contains authenticator attachment descriptions, which are used as an additional
+    - The optional <dfn>attachment</dfn> parameter contains authenticator attachment descriptions, which are used as an additional
         constraint on which authenticators are eligible to participate in a [[#makeCredential]] or [[#getAssertion]] operation.
         See [[#attachment]] for a description of the attachment values and their meanings.
 
+    - The optional <dfn>requireUserVerification</dfn> parameter can be used as an additional authenticator selection criteria. 
+        If this parameter is set to <i>true</i>, only  authenticators capable of user verification would participate in the 
+        [[#makeCredential]] operation, thereby ensuring the authenticator will perform the user verification process during 
+        the [[#makeCredential]] operation and future [[#getAssertion]] operation when it is requested to verify the credential. 
```

Comment:
You need to use this value someplace in the algorithm for makeCredential. I'm thinking somewhere around line 508.